.github/workflows/build-images.yml

```diff
@@ -0,0 +1,70 @@
+name: build-images
+
+on:
+  push:
+    branches:
+      - edge
+      - latest
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+  pull_request:
+  workflow_dispatch:
```

Comment:
I think we should have on `release` instead

Reply:
I dont mind, was just an idea